index.js

```diff
@@ -125,7 +125,7 @@ Discovery.prototype.destroy = function (cb) {
     self.tracker.removeListener('peer', self._onTrackerPeer)
     self.tracker.removeListener('update', self._onTrackerAnnounce)
     tasks.push(function (cb) {
-      self.tracker.destroy(cb)
+      self.tracker = null;
```

Comment:
It's not enough to just set `self.tracker = null` for garbage collection. There are open sockets and other objects that need to have `destroy()` called on them. So, unfortunately, this approach won't work.